src/entities/identity/model/actions.ts

```diff
@@ -1,3 +1,3 @@
-import { testIdentitySlice } from './slice';
+import { identitySlice } from './slice';
 
-export const onAuthSuccess = testIdentitySlice.setAuth;
+export const { onAuthSuccess } = identitySlice.actions;
```

Comment:
I think we can export actions like this:
`export const actions = identitySlice.actions;`
`export const reducer = identitySlice.reducer;`
or even
`export const { actions, reducer } = identitySlice`;